dev-system/resources/cors_headers_test.html

```diff
@@ -7,23 +7,30 @@
  </head>
  <body>
    <h1 id="resp"></h1>
+   <input type="text" id="scroll-id" value=""/>
+   <button onclick="submit()" id="scroll">Submit Scroll-Id</button>
    <script>
-     var req = new XMLHttpRequest();
-     var url = 'http://localhost:3003/collections.json';
-     req.open('GET', url, true);
-     req.setRequestHeader('Client-Id', 'Sprint44Demo');
-     req.setRequestHeader('CMR-Request-Id', 'sprint44-demo');
-     req.onload = function() {
-      var hits = req.getResponseHeader("CMR-Hits");
-      var requestId = req.getResponseHeader("CMR-Request-Id");
-      var response = "CMR-Hits: " + hits + "\nCMR-Request-Id: " + requestId + "\nResult: " + req.responseText;
-       document.getElementById('resp').innerText = response;
-     };
-     req.onerror = function() {
-       document.getElementById('resp').innerText = "Fail!";
-     };
-     req.send();
+     function submit() {
+       var req = new XMLHttpRequest();
+       var url = 'http://localhost:3003/collections.json?scroll=true';
+       var scrollHeader = document.getElementById("scroll-id").value;
+       req.open('GET', url, true);
+       req.setRequestHeader('Client-Id', 'Sprint44Demo');
+       req.setRequestHeader('CMR-Request-Id', 'sprint44-demo');
+       if (scrollHeader) req.setRequestHeader('CMR-Scroll-Id', scrollHeader);
+       req.onload = function() {
+         var hits = req.getResponseHeader("CMR-Hits");
+         var requestId = req.getResponseHeader("CMR-Request-Id");
+         var scrollId = req.getResponseHeader("CMR-Scroll-Id");
+         var response = "CMR-Hits: " + hits + "\nCMR-Request-Id: " + requestId + "\nCMR-Scroll-Id: " + scrollId + "\nResult: " + req.responseText;
```

Comment:
It's not obvious to me how we're verifying that we can accept the CMR-Scroll-Id header and return the header as part of a CORS request.

Is this somehow incorporated in an automated test?